src/styles/_catalog.scss

```diff
@@ -14,6 +14,13 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+
+    transition-duration: 300ms;
```

Comment:
Check the 10th point in the [checklist](https://github.com/mate-academy/layout_catalog_hovers/blob/master/checklist.md)